src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-form/toolbar-form-fields.tsx

```diff
@@ -87,22 +86,23 @@ const ToolbarFormFields = ({
     });
   };
 
+  //FIXME: Implement vacation days calculations properly
   /**
    * Get a list of working days
    *
-   * @param loggedInPerson Person
+   * @param loggedInUser User
    */
-  const getWorkingWeek = (loggedInPerson?: Person) => {
-    const workingWeek = new Array(DAYS_OF_WEEK.length).fill(false);
-    if (!loggedInPerson) return workingWeek;
-
-    DAYS_OF_WEEK.forEach((weekDay, index) => {
-      if (loggedInPerson[weekDay as keyof typeof loggedInPerson] !== 0) {
-        workingWeek[index] = true;
-      }
-    });
-    return workingWeek;
-  };
+  // const getWorkingWeek = (loggedInUser?: User) => {
+  //   const workingWeek = new Array(DAYS_OF_WEEK.length).fill(false);
+  //   if (!loggedInUser) return workingWeek;
+  //
+  //   DAYS_OF_WEEK.forEach((weekDay, index) => {
+  //     if (loggedInUser[weekDay as keyof typeof loggedInUser] !== 0) {
+  //       workingWeek[index] = true;
+  //     }
+  //   });
+  //   return workingWeek;
+  // };
```

Comment:
I think it's unlikely we will track a persons working week in the same way in the future, if you agree feel free to remove this commented out code and the related function call